src/opnsense/mvc/app/models/OPNsense/IPsec/Menu/Menu.xml

```diff
@@ -6,12 +6,12 @@
                 <Phase1 url="/vpn_ipsec_phase1.php*" visibility="hidden"/>
                 <Phase2 url="/vpn_ipsec_phase2.php*" visibility="hidden"/>
             </Tunnels>
-            <Mobile order="20" VisibleName="Mobile Clients" url="/vpn_ipsec_mobile.php">
+            <Mobile order="20" VisibleName="Mobile Clients (legacy)" url="/vpn_ipsec_mobile.php">
                 <Act url="/vpn_ipsec_mobile.php*" visibility="hidden"/>
             </Mobile>
             <Keys order="30" VisibleName="Pre-Shared Keys" url="/ui/ipsec/pre_shared_keys/"/>
             <KeyPairs order="40" VisibleName="Key Pairs" url="/ui/ipsec/key_pairs" />
-            <Settings order="50" VisibleName="Advanced Settings" url="/ui/ipsec/connections/settings"/>
+            <Settings order="50" VisibleName="Mobile &amp; Advanced Settings" url="/ui/ipsec/connections/settings"/>
```

Comment:
This has an encoding issue in the menu search field, noticed with Dnsmasq but forgot to mention

Reply:
@fichtner thanks, I'll check on my end, rather fix the search then remove the ampersand